py/desisim/scripts/quickquasars.py

```diff
@@ -55,7 +56,13 @@ def parse(options=None):
     parser.add_argument('--wmax', type=float, default=10000,help="Max wavelength (obs. frame)")
     parser.add_argument('--dwave', type=float, default=0.2,help="Internal wavelength step (don't change this)")
     parser.add_argument('--nproc', type=int, default=1,help="number of processors to run faster")
-    parser.add_argument('--zbest', action = "store_true" ,help="add a zbest file per spectrum with the truth")
+    
+    parser.add_argument('--zbest', action = "store_true",help="add a zbest file per spectrum either with the truth redshift or adding some error (use --sigma_kms_rsd and/or --sigma_kms_zfit)")
+
+    parser.add_argument('--sigma_kms_rsd',nargs='?',type=float,const=150, help="Adds a gaussian error to the quasar redshift thats simulate the fingers of god effect (default=150 km/s)")
```

Comment:
I understood we want that if --zbest option is used alone, it will write out the truth redshift to the zbest file. Now, if --sigma_kms_rsd is used without argument it will use sigma_kms_rsd=150 as default. It is the same as default but only if the  --sigma_kms_rsd argument  is present, if you set, for instance,   --sigma_kms_rsd  200, then it will use 200...

Reply:
Oh, I see, good to know this is possible to do!

But that's why in the comments below I wanted to distinguish the two type of "redshift errors". While we want to make --sigma_kms_fit an option that we only add when we will not run Redrock afterwards, I'd say that we always want to add --sigma_kms_rsd, so I would set it to 150 by *default*, so that we don't need to always explicitely run with the option.

Only in very special cases, when debugging the code or similar, we might want to not add fingers of god, but in that case we can just do --sigma_kms_rsd 0.0

As I mention in the comments below, it might be better if we use instead --sigma_kms_fog, to make it less confusing (RSD redshift is another option in the transmission file).

Reply:
OK! got  it...